README.rst

```diff
@@ -0,0 +1,60 @@
+cc2olx
+######
+
+A hackathon project to explore importing Common Cartridge courses into Open edX
```

Comment:
Could we update this section since we're filling out the features of this and building into more than just a hackathon project? :)